### ISSUE TITLE
Gfycats that are reuploaded do not have audio.

### ISSUE THREAD

**Issue by nmur** [nmur/reddit-video-rotation-bot#27]
Looks like this is because the downloaded videos don't have audio in the first place.

For a media URL like 
```
https://v.redd.it/k3zyrnirbb651/DASH_1080?source=fallback
```
we can get the audio by replacing `DASH_1080?...` with `audio`
```
https://v.redd.it/k3zyrnirbb651/audio
```
Which would enable us to download the audio as well, and then hopefully add the audio to the video with some ffmpeg experimentation. Then we should be able to proceed with video as normal.

**Comment by nmur** [comment]
Looks like we can take both the video and the audio files and combine them with minimal (none?) encoding with a command like:
```sh
ffmpeg -i video.mp4 -i audio.mp4 -c copy output.mkv
```

**Comment by nmur** [comment]
Also it appears that not every media post has audio (eg. Gfycat posts), so we shouldn't design the implementation so that it depends on audio being there.

**Comment by nmur** [comment]
Looks like the DASH video/audio links can vary between posts for some reason. The DASHPlaylist endpoint that is available in the .json representation of the post can be downloaded and viewed in a text editor to view all of the video/audio endpoints available. 

For video, parsing this out isn't required because we're also supplied with the provided video stream endpoint. However for audio, I've already seen two different audio endpoints (`/audio`, and `/DASH_audio.mp4`). So we either need to try both of those possibilities, or go down the path of downloading and parsing the DASHPlaylist.mpd XML doc to find the audio. We'll take the first approach for now, but will have to adapt the more generic approach if any more variations are found.

**Comment by nmur** [comment]
Resolved by #38, #39, and #40.
